ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -332,6 +332,14 @@ tests:
         NUM_MASTERS=1
       TEST_SUITE: minimal
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-static-ip
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        IS_STATIC_IP=true
```

Comment:
Can we rename the env var so signal which code base it refers to?

Reply:
Which env-var? ASSISTED_CONFIG?

Reply:
IS_STATIC_IP

Reply:
But it's under ASSISTED_CONFIG
Also all other env-vars are not indicative about their source, NUM_MASTERS, IPV4, etc.